lib/bundle.js

```diff
@@ -103,31 +199,33 @@ exports.bundle = function(moduleExpression, fileName, opts) {
     return systemBuilder.trace(firstModule, cfg);
```

Comment:
@ahmedshuhel I'd still suggest that we glob in the module name space. It is important to allow users to be able to do `github:components/jquery@1.2.3/**/*` so that they don't write `jspm_packages/github/components/jquery@1.2.3/**/*`.

The worry of which syntax to allow is still valid here. So I'd still suggest creating a list of characters to escape.